lib/HLSL/DxilCondenseResources.cpp

```diff
@@ -120,7 +120,7 @@ class DxilResourceRegisterAllocator {
 
   template <typename T>
   static bool
-  AllocateRegisters(const std::vector<std::unique_ptr<T>> &resourceList,
+  AllocateRegisters(DxilModule &DM, const std::vector<std::unique_ptr<T>> &resourceList,
```

Comment:
See comment about not needing DxilModule in dxilutil error message function.  With that, and the fact that `Ctx` is now unused, we can get rid of both of these arguments.